docs/admin/system_admin/load_balancing.md

```diff
@@ -0,0 +1,71 @@
+
+Activating either of these features (reusing connection to backends or tunnels with keep-alive to both client and backend) can significantly reduce load and increase performance compared to establishing connection between LB and backend for every request. However, the downside is that memory requirements for LB are increased and some extra tweaking might be needed. The most important tweak is keep-alive timeout between LB and backend. LeoGateway has timeout parameters (`http.timeout_for_header` and `http.timeout_for_body`), first of which is set to 5s by default. It means that keep-alive connection without any traffic in it for 5 seconds will be closed by LeoGateway. It's a good idea to increase this parameter when operating in the above modes. Regardless whether it's increased or not, the similar parameter on LB side should be set to **lower** value than timeout on LeoGateway. In HAProxy, this is controlled by `timeout http-keep-alive` in the backend section. This is especially important when HTTP connections are reused, because it avoids the problem when LB tries to send data into existing connection at the same time as LeoGateway closes that connection. Ensuring that LeoGateway keeps the inactive connection open a bit longer than the time after which LB will close connection from its side solves this problem.
+
+**SSL/TLS termination**: obviously, for LB to work with LeoGateway in HTTP protocol mode, SSL has to be terminated on LB itself. Usually (e.g. in case of HAProxy in multi-process mode) it will provide much higher SSL performance compared to serving SSL from LeoGateway directly, so LeoGateway should always be configured to use HTTP when operating behind LB. In cases where secure traffic between LB and LeoGateway is a must, either LB should be configured to decrypt/re-encrypt SSL traffic (so it can still operate in HTTP and not in TCP tunneling mode), or some other layer of encryption, like VPN should be used.
+
+**High availability**: a single load balancer becomes a point of failure (PoF) itself. For highly available configurations, at least two LBs are needed, configured for failover (e.g. using floating IP with Corosync / Pacemaker). Alternatively, IP-level LB (e.g. Linux Virtual Server) can be set up in front of HTTP-level LBs. IP-level balancing solutions provide higher availability compared to failover and can distribute load between multiple HTTP-level LBs, but require dedicated hardware.
```

Comment:
Add the **Related Links** section for
- http://leo-project.net/leofs/docs/admin/settings/leo_gateway/
- https://cbonte.github.io/haproxy-dconv/1.6/configuration.html#2

Just in case, there is another format convention for external links (the link for haproxy in this case) so please follow the format used at the link for Amazon S3 API https://github.com/leo-project/leofs/blob/develop/docs/admin/settings/leo_gateway.md#related-links